apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -127,7 +127,9 @@ const ChangeProjectDomain = ({
       }
       suffix={
         <Grid flow="column" align="center">
-          <Tooltip content={error !== undefined ? error : statusText}>
+          <Tooltip
```

Comment:
if content text is string, its already being rendered as Text:
`              {typeof content === "string" ? <Text>{content}</Text> : content}` in Tooltip